packages/airnode-deployer/terraform/airnode/gcp/main.tf

```diff
@@ -40,7 +43,8 @@ module "startCoordinator" {
   name               = "${local.name_prefix}-startCoordinator"
   entry_point        = "startCoordinator"
   source_dir         = var.handler_dir
-  timeout            = 60
+  memory_size        = 256
+  timeout            = 65
```

Comment:
65 seconds?

Reply:
The sum of the timeouts of the non-`startCoordinator` handlers for this PR are 59s, so my idea was to give `startCoordinator` a few seconds extra to collect the result(s) from `processProviderRequests`, log execution time and also report a heartbeat.

The timeout could be more like 61 or 62 seconds, but there didn't seem any harm in giving it extra headroom.

If you disagree I can reduce the timeouts on the other handlers slightly to allow the heartbeat to be reported.

Reply:
Yeah I think that's _probably_ fine. This will also be slightly different for non-serverless deployments then, since they will still only be allowed 60 seconds. Maybe that's an issue, maybe not - I'm not sure

Reply:
> since they will still only be allowed 60 seconds.
When you say "non-serverless", you mean `yarn run-airnode-locally`?
If so, that's just a cron job, so it won't be restricted, but I'm happy to test.

Reply:
I tested the local variant and it does introduce a problem (Airnode runs every two minutes instead of every minute).
I'll produce a fix for this shortly (by forking Airnode into a child process rather than blocking Cron, which is better behaviour imo anyway).

Reply:
Done - cron now doesn't wait for Airnode to finish executing.